docs/package.json

```diff
@@ -31,7 +31,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/react-fontawesome": "^0.1.11",
     "@material-ui/core": "5.0.0-alpha.34",
-    "@material-ui/data-grid": "^4.0.0-alpha.21",
+    "@material-ui/data-grid": "^4.0.0-alpha.29",
```

Comment:
This is needed, as this version does not use `makeStyles` from `@material-ui/core/styles` anymore.